W-17248489-Fixing type coercion examples to match current behavior (#420)
mulesoft/docs-dataweave: modules/ROOT/pages/dataweave-types-coercion.adoc

"08/30/2020"
----

> The following example converts a number to a string with decimal representation.

Reviewer: ```suggestion
This example converts a number to a string with a decimal representation:
```

[source,json,linenums]
----
{
  "data": "22.00"
}
----

[NOTE]

> To maintain consistency across all formats, starting with DataWeave 2.5, writing number values directly doesn’t preserve formatting and strips trailing zeroes. Format metadata is only used to parse the provided string into the correct number representation. To retain this behavior, you can re-enable it by using the `stripTrailingZeroes` property. See xref:dataweave-system-properties.adoc[supported data formats]

Reviewer: ```suggestion
To maintain consistency across all formats, starting with DataWeave 2.5, writing number values directly doesn’t preserve formatting and strips trailing zeroes. Format metadata is only used to parse the provided string into the correct number representation. To retain this behavior, you can re-enable it by using the `stripTrailingZeroes` property. See xref:dataweave-system-properties.adoc[].
```

> Note: In order to be consistent across all formats, since DataWeave 2.5 writing number values directly will not preserve format and will strip trailing zeroes. Format metadata will only be used to parse the provided string into the right number representation. If you still need this behavior you can re-enable it by using the `stripTrailingZeroes` property found here:
> xref:dataweave-system-properties.adoc[Supported Data Formats]

Reviewer: ```suggestion
[NOTE]
To maintain consistency across all formats, starting with DataWeave 2.5, writing number values directly doesn’t preserve formatting and strips trailing zeroes. Format metadata is only used to parse the provided string into the correct number representation. To retain this behavior, you can re-enable it by using the `stripTrailingZeroes` property. See xref:dataweave-system-properties.adoc[supported data formats]
```
